crates/ruff_rustpython/src/lib.rs

```diff
@@ -20,8 +20,14 @@ pub fn tokenize(contents: &str) -> Vec<LexResult> {
 pub fn parse_program_tokens(
     lxr: Vec<LexResult>,
     source_path: &str,
+    is_jupyter_notebook: bool,
 ) -> anyhow::Result<Suite, ParseError> {
```

Comment:
The reason to use a `bool` here instead of `Mode` is that the function is expected to return a `Module` node which is only possible for 2 (`Module`, `Jupyter`) out of the 4 modes.